lib/init.coffee

```diff
@@ -27,22 +16,25 @@ module.exports =
       grammarScopes: @scopes
       scope: 'file'
       lintOnFly: true
-      lint: (textEditor) ->
+      lint: (textEditor) =>
```

Comment:
What's the difference between -> and => (I'm just curious about it, I'll google it later).